langfuse/integrations.py

```diff
@@ -0,0 +1,83 @@
+import os
+import functools
+from datetime import datetime
+from dotenv import load_dotenv
+
+import openai
+from openai.api_resources import ChatCompletion, Completion
+
+from langfuse import Langfuse
+from langfuse.client import InitialGeneration
+from langfuse.api.resources.commons.types.llm_usage import LlmUsage
+
+
+load_dotenv()
+
+
+class OpenAILangfuse:
```

Comment:
I would maybe rename the class to reflect a bit better of how it works.

Reply:
Do you suggest any?

Reply:
Sure: `LangfuseOpenAiWrapper`

Reply:
WDYT?

Reply:
Sounds good